symplyphysics/laws/electricity/circuits/resistivity_of_parallel_resistors.py

```diff
@@ -0,0 +1,36 @@
+from sympy import Eq, solve
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+from symplyphysics.core.operations.sum_array import SumArray
+from symplyphysics.core.symbols.symbols import tuple_of_symbols
+
+# Description
+## If resistors are connected in parallel, the inverse of total resistance is the sum of inverse resistance of each resistor.
+## Law: 1/R_parallel = sum(1/R[i]), where
```

Comment:
Maybe it is better to write it for total resistance on the left side? I do not see, why we should have it inverted.